Makefile

```
@@ -60,6 +60,11 @@ ENVTEST_K8S_VERSION = 1.31
 SHELL = /usr/bin/env bash -o pipefail
 .SHELLFLAGS = -ec
 
+# Scaler schema generation parameters
+ SCALERS_BUILDER_FILE ?= "pkg/scaling/scalers_builder.go"
+ SCALERS_FILES_DIR ?= "pkg/scalers"
+ OUTPUT_FILE_PATH ?= "schema/"
```

```suggestion
 OUTPUT_FILE_PATH ?= "schema/generated"
```

Maybe this? Not sure if needed though